app_dart/lib/src/request_handlers/check_for_waiting_pull_requests_queries.dart

```diff
@@ -23,27 +23,6 @@ query LabeledPullRequcodeestsWithReviews($sOwner: String!, $sName: String!, $sLa
                   oid
                   committedDate
                   pushedDate
-                  status {
-                    contexts {
-                      context
-                      state
-                    }
-                  }
```

Comment:
Don't we still need this?

Reply:
I doubt. Both the removed checkSuites and statuses below were used for cirrus status check if I understand correctly. After using cirrus API to get and calculate the status, we do not seem to need any of those two.
```final List<Map<String, dynamic>> checkSuites =
          commit['checkSuites']['nodes'].cast<Map<String, dynamic>>();
      final List<Map<String, dynamic>> statuses =
          commit['status']['contexts'].cast<Map<String, dynamic>>();```

Reply:
The status check is needed to see if the tree is greeen for luci-engine and flutter-build

Reply:
Sorry I missed that part. Now added statuses back.